includes/class-vuln-service.php

```diff
@@ -14,6 +14,11 @@
  */
 class Vuln_Service {
 
+	/**
+	 * @var array Array of command arguments.
+	 */
+	private $assoc_args;
```

Comment:
Thanks for adding this. As we are using this in child classes, it should be `protected`.
```suggestion
	protected $assoc_args;
```

Reply:
Done 🍏